inc/templates/post-list-single-1-cols.php

```diff
@@ -44,9 +44,9 @@
 			<?php echo_post_meta($post,array('date','sources'),$order); ?>
 		<?php endif; ?>
 
-		<?php if ($show_summary_translated_by_odc_team): ?>
+		<!-- <?php if ($show_summary_translated_by_odc_team): ?>
```

Comment:
@S-mardii please do not leave commented code behind. If we do not need this anymore then it should be removed.